taxcalc/policy_current_law.json

```diff
@@ -745,14 +753,14 @@
                   [9e99, 9e99, 9e99, 9e99, 9e99],
                   [9e99, 9e99, 9e99, 9e99, 9e99],
                   [250000.0, 500000.0, 250000.0, 250000.0, 500000.0],
-                  [255600.0, 511200.0, 255600.0, 255600.0, 511200.0],
```

Comment:
@Peter-Metz, do you know why the 2019 values here appeared to be rounded already?

Reply:
Peter confirmed over chat that the previous years are correct, so I am going to leave this.